demo/file_component/run.py

```diff
@@ -1,8 +1,7 @@
-import gradio as gr 
+import gradio as gr
 
-css = "footer {display: none !important;} .gradio-container {min-height: 0px !important;}"
 
-with gr.Blocks(css=css) as demo:
+with gr.Blocks() as demo:
     gr.File()
 
```

Comment:
How come we only removed the `css` from this component, wheareas all of the other `*_component` demos still include this css?

Reply:
That was a leftover, will revert. We can remove some of these custom things now but I'll do that in a follow-up to reduce noise.